### PR TITLE
Add optional statistics collection

### PR DESCRIPTION
**Pull request by 8ntexerra**
You can now pass props if you want to collector starts, it's disabled by default.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -121,22 +122,33 @@ export default class VideoPlayer extends Component {
...
-    stats.deactivate()
+    this.deactivateStatistics()
     if (this.controlsTimeout) {
       clearTimeout(this.controlsTimeout);
       this.controlsTimeout = null;
     }
   }
 
+  deactivateStatistics () {
+    this.state.stats && this.deactivateStatistics()
```
Comment (YuriBely) [inline index.js]: @8ntexerra  are you sure about this? I suggest here is should be stats.deactivate method

File: index.js [hunk 2]
```diff
@@ -121,22 +122,33 @@ export default class VideoPlayer extends Component {
...
+  }
+
+  collectStatistics () {
+    this.state.stats && this.state.isPlaying && stats.add(this.playerMode).activate()
```
Comment (YuriBely) [inline index.js]: Is here really need check isPlaying state? In code this method called after when isPlaying state changed.

**Review by YuriBely — COMMENTED** [review]
Please follow the style guide for current form. I think you used some autoformatter tools (or IDE) and it brake the code style. So please clean up.

File: index.js (hunk 2 above)
Comment (8ntexerra) [reply index.js]: There is one condition when it's called on duration changes
`
this.setState({ duration }, () => this.state.isPlaying && !this.wasPlayingBeforeSeek
      ? stats.add(this.playerMode).activate()
      : null
    )
`

But in the last commit I merged all checks in one function

**Comment by YuriBely** [comment]
@8ntexerra Much better now. Merged

**Pull request merged**